openid-connect-server/src/main/java/org/mitre/oauth2/web/ScopeAPI.java

```diff
@@ -101,7 +105,14 @@ public String updateScope(@PathVariable("id") Long id, @RequestBody String json,
 		SystemScope existing = scopeService.getById(id);
 
 		SystemScope scope = gson.fromJson(json, SystemScope.class);
-
+		try {
+			validateScope(scope);
+		} catch (ScopeException e) {
+			logger.error("updateScope failed due to ScopeException. {}", e.getMessage());
```

Comment:
Could we add more information regarding what is that we are trying to change, id for example?

Reply:
OK, I can see the invalid scope is part of the ScopeException

Reply:
Just confirming you're happy for me to resolve this - do you wish for any further information on these error messages?

Reply:
I think it's OK. At least it contains the scope that failed on the exception.

Reply:
Sorry for being annoying, can we pass the exception and not exception.getMessage() to the logger please? I can see you are already doing that on the ClientAPI.

Reply:
Yep, makes sense